talaiot/src/main/kotlin/com/cdsap/talaiot/metrics/TaskMetrics.kt

```diff
@@ -0,0 +1,47 @@
+package com.cdsap.talaiot.metrics
+
+import com.cdsap.talaiot.entities.CacheInfo
+import com.cdsap.talaiot.metrics.base.ExecutedTasksMetric
+
+class CacheHitMetric : ExecutedTasksMetric<BuildCacheUsageStats>(
+    provider = { info ->
+        var localCacheHitCount = 0
+        var localCacheMissCount = 0
+        var remoteCacheHitCount = 0
+        var remoteCacheMissCount = 0
+        info.executedTasksInfo.values.forEach {
+            when (it.localCacheInfo) {
+                is CacheInfo.CacheHit -> localCacheHitCount++
+                is CacheInfo.CacheMiss -> localCacheMissCount++
+                CacheInfo.CacheDisabled -> {
+                }
+            }
+
+            when (it.remoteCacheInfo) {
```

Comment:
Nice! I have seen cases when there is bad connectivity Gradle disabled the cache mechanism. Is covered in the when +1